pkg/ephemeral-disk-utils/utils.go

```diff
@@ -96,7 +96,7 @@ func Md5CheckSum(filePath string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer file.Close()
+	defer file.Close() // #nosec no risk for readonly files
```

Comment:
I always disliked these inline comments that come to satisfy an external tool.
It makes the code ugly, requiring me to read it when I walk through the code and distracts me.

There may be dozens on these all over now, forcing us to pollute the code with endless comments like these.
BTW, if someone adds a new defer like this tomorrow, or changes this one to a rw file instead. How will you track it?

Reply:
@EdDev I think the plan is to clean all gosec issues, and then add gosec to the regular build & PR tests so that a developer would be able to directly fix the gosec issues his code introduced. 
Regarding the comments.  I agree they might pollute the code, but I'm not sure what do you suggest should be the alterative. As I see it there are two option 1) disable a test globally 2) Review each potential violation and "ignore" it manually through this comment. 
Judging from the code I have already reviewed I think we should better go with (2) then disabling the test and then chasing possible (some times very hard) bugs.

Reply:
I agree this need to be generally discussed. Not only some of these are not handled in large projects , some are not even used in core libraries of golang itself !